tests/networkpolicy_test.go

```diff
@@ -1,173 +1,234 @@
 package tests_test
 
 import (
+	"fmt"
 	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	v1network "k8s.io/api/networking/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-	v13 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	corev1 "k8s.io/api/core/v1"
+	networkv1 "k8s.io/api/networking/v1"
+	errors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	netutils "k8s.io/utils/net"
 
 	v1 "kubevirt.io/client-go/api/v1"
 	"kubevirt.io/client-go/kubecli"
 	"kubevirt.io/kubevirt/tests"
 	cd "kubevirt.io/kubevirt/tests/containerdisk"
+	"kubevirt.io/kubevirt/tests/libvmi"
 )
 
 var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:component]Networkpolicy", func() {
 
-	var virtClient kubecli.KubevirtClient
+	var (
+		virtClient                                          kubecli.KubevirtClient
+		serverVMI, clientVMI, clientVMIAlternativeNamespace *v1.VirtualMachineInstance
+		serverVMILabels                                     = map[string]string{"type": "test"}
+	)
+	Context("when three cirros VMs with default networking are started and serverVMI start an HTTP server on port 80 and 81", func() {
+		tests.BeforeAll(func() {
+			var err error
 
-	var vmia *v1.VirtualMachineInstance
-	var vmib *v1.VirtualMachineInstance
-	var vmic *v1.VirtualMachineInstance
+			virtClient, err = kubecli.GetKubevirtClient()
+			tests.PanicOnError(err)
 
-	tests.BeforeAll(func() {
-		var err error
+			tests.SkipIfUseFlannel(virtClient)
+			skipNetworkPolicyRunningOnKindInfra()
+			tests.BeforeTestCleanup()
```

Comment:
The `tests.BeforeTestCleanup` remove in our case VMIs and test namespces, the network policies are removed at every `AfterEach` so is safe to call it from here since we create the VMIs only once too.

Reply:
Ohh now I see... We just are leaving the VMI/s without cleaning them and hope the next test will clean it up.